kubeflow/trainer/backends/container/backend.py

```
@@ -800,6 +806,39 @@ def get_job_logs(
+                status, exit_code = self._adapter.container_status(container["id"])
+                lines.append(f"  {step}: status={status}, exit_code={exit_code}")
+                if status == "exited" and exit_code != 0:
+                    try:
+                        tail: deque[str] = deque(maxlen=10)
+                        for chunk in self._adapter.container_logs(container["id"], follow=False):
+                            tail.extend(chunk.strip().splitlines())
+                        if tail:
+                            lines.append("  Last logs:")
+                            for log_line in tail:
+                                lines.append(f"    {log_line}")
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)